packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -98,3 +196,33 @@ blockquote {
   border-right: 1px solid cv("gray", "300");
   border-radius: 4px;
 }
+
+textarea::-webkit-input-placeholder,
+input::-webkit-input-placeholder {
+  font-family: "Aeonik";
+}
+
+textarea::-moz-placeholder,
+input::-moz-placeholder {
+  font-family: "Aeonik";
+}
+
+textarea:-moz-placeholder,
+input:-moz-placeholder {
+  font-family: "Aeonik";
+}
+
+textarea:-ms-input-placeholder,
+input:-ms-input-placeholder {
+  font-family: "Aeonik";
+}
+
+button {
+  background: transparent;
+
+  font-family: "Aeonik";
+}
+
+a[href="#"] {
+  font-weight: 500 !important;
+}
```

Comment:
Is this declaration necessary when we already have the other one for `a` above on line 133?

Reply:
no, we don't. deleted line